src/plays/text-avatar/components/UserCard/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { User } from '../../types';
+import Avatar from '../Avatar';
+import './style.scss';
+type Props = {
+    user: User
+}
+function UserCard({ user }: Props) {
+    const { first, last } = user.name;
```

Comment:
Can we use User Type directly like this?
```suggestion
function UserCard({ first, last }: User) {
```

Reply:
I think I should keep it like that since me or others can contribute to it and need to provide more props

Reply:
@hiimnhan , If we need then we can but for now we can directly use the User type.